git_hashes.py

```diff
@@ -118,5 +133,7 @@ def get_hash_status(h):
 
 	hashes = [ h for h in hashes if h ] # filter any null hashes
 
+	hashes = [h for h in hashes if check_ocaml_version_mismatch(args.sandmark_tag_override, h)]
```

Comment:
This will fail if `sandmark_tag_override` is not set. (`sandmark_tag_override` is not set for `run_backfill.py` and defaults to `None` in `run_sandmark_backfill.py`)

Reply:
With commit `127c30...`:
1. In run_backfill.py, args.sandmark_tag_override is set to None, since it is not one of the input parameters.
2. In git_hashes.py, we now check if args.sandmark_tag_override is provided before applying the filter.